GDJS/Runtime/pixi-renderers/pixi-filters-tools.js

```diff
@@ -1,5 +1,8 @@
 gdjs.PixiFiltersTools = function() {};
 
+const clampValue = (value, min, max) => Math.max(min, Math.min(max, value));
+const clampKernelSize = value => ([5, 7, 9, 11, 13, 15].includes(value)) ? value : 5;
```

Comment:
Same for this function. 
`includes` is also fairly recent and could create issue with old browser that we still support (read: Android 4.x). Use indexOf instead: `[5, 7, 9, 11, 13, 15].indexOf(value) !== -1`

Reply:
Note that in newIDE, we're running with Babel, so all the latest features of JS can be used, and using webpack to bundle JS, so declare function/variables anywhere is fine because they will be scoped to the JS file - no risk of collision.